frontend/components/analytics/GoogleAnalytics.tsx

```diff
@@ -22,8 +22,22 @@ export function GoogleAnalytics() {
       };
    }, [router?.events]);
 
-   return GA_URL ? (
+   return GA_URL || GTAG_ID ? (
       <>
+         <Script
+            id="gtag-ga4"
+            strategy="afterInteractive"
+            src={`https://www.googletagmanager.com/gtag/js?id=${GTAG_ID}`}
```

Comment:
If `GTAG_ID` is falsy, then don't we want to *not* include this script tag?

As written, this will always include the tag if either the `GA_URL` OR `GTAG_ID` is set.

Reply:
Fair point. Do we want to treat them as independent or both required?